### ISSUE TITLE
Flatpak Support

### ISSUE THREAD

**Issue by SeongGino** [SeongGino/Nero-umu#10]
I've never actually looked at or done anything with Flatpak outside of using some (select few) apps through it, so I'm not sure what the requirements of Flatpak support are.

FWIW: regarding filesystems, we rely currently on a couple of things that are found in the local filesystem:
 - `umu-run` itself, because *duh*
 - `curl` for external downloading (currently only used for downloading the Discord Bridge service, but will be used elsewhere), waiting on another issue to pack this in as a build dependency rather than runtime.
 - `~/.steam/steam/compatibilitytools.d` for finding existing Steam Proton runners - not completely necessary, as Nero Manager can simply make its own directory that it uses to store runners.
 - `~/.config` for the manager configuration file, which stores the prefixes path and basic settings for the frontend.
 - Whatever gets defined as the prefixes "Home"
 - `/tmp`, for storing either downloads (currently only Discord Bridge) or icons extracted from applications.

Nero Manager also provides a system tray icon, but I *assume* Flats should be able to use these just fine?

I have to assume, due to sandboxing, that any access to filesystems needs to go through some kind of filter or some shit?
